packages/ai-bot/helpers.ts

```diff
@@ -570,6 +574,16 @@ export function isCommandEvent(
   );
 }
 
+function getModel(eventlist: DiscreteMatrixEvent[]): string {
+  let activeLLMEvent = eventlist.findLast(
```

Comment:
Should we be reading this from room state instead?

Reply:
We could read this from the room state, but I chose to read it from the event list to make testing easier. Additionally, this approach is consistent with how we read skills from the room.